src/org/opendatakit/briefcase/ui/automation/AutomationPanelForm.java

```diff
@@ -25,8 +24,7 @@
   private JTextField scriptDirField;
   private JButton scriptDirChooseButton;
   private JButton generateScriptButton;
-
-  private List<Consumer<Path>> onSelectScriptDirCallbacks = new ArrayList<>();
+  private AutomationConfiguration automationConfiguration = AutomationConfiguration.empty();
```

Comment:
Having a conf object as a member of the view is how we solved this problem in the `ConfigurationPanel`, but it has turned out to be much better to build the object ad-hoc just before calling the callbacks in the action listener of the generate script button.